src/bp-moderation/classes/class-bp-rest-moderation-report-endpoint.php

```diff
@@ -500,6 +500,12 @@ protected function prepare_report_link( $item_id, $item_type ) {
 					'embeddable' => true,
 				);
 				break;
+			case BP_Suspend_Video::$type:
```

Comment:
@janakkaneriya007 move this change to PR: https://github.com/buddyboss/buddyboss-platform-api/pull/284

Reply:
https://github.com/buddyboss/buddyboss-platform-api/pull/288